CODING_GUIDELINES.md

```diff
@@ -232,36 +232,69 @@ class MyClass
 ```
 
 ### 7. Header inclusion order. ###
-The headers should be placed in the following order:
- 1. Module header (in .cpp)
- 2. System/Qt/Boost etc. headers (split in subcategories if you have many).
- 3. Application headers, starting from *Base* headers.
+The headers should be placed in the following group order:
+  1. Module header (in .cpp)
+  2. C++ Standard Library headers
+  3. System headers
+  4. Boost library headers
+  5. Libtorrent headers
+  6. Qt headers
+  7. qBittorrent own headers, starting from *base* headers.
+
+The headers should be ordered alphabetically within each group.  
+If there are conditionals for the same header group, then put them at the bottom of the respective group.  
+If there are conditionals for the different header groups, then put them above of "qBittorrent own headers" group.
```

Comment:
@Chocobo1:  **Grammar** **alert:**  

> above **the** "qBittorrent own headers" group.  

(Due to the odd way English handles quotation marks in proper-nouns.)